### PR TITLE
Issue966 fast hvac storage

### LINKED ISSUE #966: AixLib.FastHVAC.Components.Storage.HeatStorageVariablePorts: Parameters not propagated
### Describe the bug
Parameters of `top_cover`, `storage_mantle[n]`, and `bottom_cover` are not propagated.

This is for the two models:
- AixLib.FastHVAC.Components.Storage.HeatStorage
- AixLib.FastHVAC.Components.Storage.HeatStorageVariablePorts

Is the HeatStorage without variable ports even still necessary?

### Approach to solve issue
1. Propagate parameters of `data` according to `AixLib.Fluid.Storage.BufferStorage`

### PR DESCRIPTION
**Pull request by DaJansenGit**
closes #966

### CODE REVIEW

File: AixLib/Obsolete/Year2020.mo [hunk 1]
```diff
@@ -0,0 +1,765 @@
...
+            smooth=Smooth.None));
+        connect(port_HC2_in, heatingCoil2.enthalpyPort_a1) annotation (Line(
+            points={{-100,-40},{-72,-40},{-72,-46.56}},
+            color={176,0,0},
+            smooth=Smooth.None));
+        connect(port_HC2_out, heatingCoil2.enthalpyPort_b1) annotation (Line(
+            points={{-100,-80},{-72,-80},{-72,-72.88}},
+            color={176,0,0},
+            smooth=Smooth.None));
+        annotation (
```
Comment (PMehrfeld) [inline AixLib/Obsolete/Year2020.mo:398]: add something like 
`obsolete = "Obsolete model - a description, e.g. which model to use instead". ` according to `AixLib.Obsolete.UsersGuide`

File: AixLib/Obsolete/Year2020.mo [hunk 2]
```diff
@@ -0,0 +1,765 @@
+within AixLib.Obsolete;
```
Comment (PMehrfeld) [inline AixLib/Obsolete/Year2020.mo:1]: Do not store as one file!

Comment (DaJansenGit) [reply AixLib/Obsolete/Year2020.mo]: solved with 27648c7888c6263f769b34c2e45c84c46dbafe42

File: AixLib/Obsolete/Year2020.mo (hunk 1 above)
Comment (DaJansenGit) [reply AixLib/Obsolete/Year2020.mo]: solved with 6cb8f61d099b0b5a1a8147c6c9e0125c12b6ee67

**Comment by PMehrfeld** [comment]
@DaJansenGit : I'll merge this. You addressed all points from above but did not re-request me for review. I just did this by myself and approved afterwards ;)

**Comment by DaJansenGit** [comment]
Oh forgot to re-request the review. Thanks for merging!

**Pull request merged**